### PR TITLE
Feature/mst account

### PR DESCRIPTION
**Pull request by Kron1749**
### **Description**

- **What does this PR do?**
 Added functions to
1.  Create MST account
2. Change name MST account
3. Delete MST account
4. Switch between MST and default account

- **Why is this change needed?**
Users want MST

### **Type of change**

- [ ] Bug fix
- [x] New feature
- [ ] Refactor (cleaning up code without changing functionality)
- [ ] Documentation update
- [ ] Tests
- [ ] Other (describe below):

---

### **Checklist**

- [ ] Code adheres to coding guidelines and standards.
- [ ] Linting and formatting checks have passed, no local issues.
- [ ] Existing tests pass successfully, no local errors.
- [ ] Pull request is linked to a relevant issue or task.
- [ ] Tests are written for new features or fixes.
- [ ] Documentation has been updated (if applicable).

---

### **Related Issue/Task**

- Issue/Task link: _[[As a user, I'd like to use MST](https://soramitsu.atlassian.net/browse/PW-1743)]_

---

### **Testing**

- **How has this been tested?**
  - _Describe manual testing steps or link to relevant automated tests._
- **Edge cases covered:**
  - _List any specific cases handled by this PR._

---

### **Screenshots/Video/Link to dedicated environment (if applicable)**

- _Include any visual examples of the change + dedicated environment (if there is a sort of automation)._

---

### **Additional Notes**

- _Add any information the reviewer should consider, especially cross-project impacts, UX changes, or potential risks._

---

### **Reviewer Tasks**

- [ ] Review logic for correctness and clarity.
- [ ] Verify that the code is following best practices.
- [ ] Test or review related areas if this PR affects other projects.

### CODE REVIEW

File: packages/sdk/src/apiAccount.ts [hunk 1]
```diff
@@ -813,4 +813,43 @@ export class ApiAccount<T = void> extends WithAccountHistory implements ISubmitE
...
+    const multisigAccount = multisigAccounts.find((account) => {
+      const accountAddress = this.formatAddress(account.address, false);
+      const targetAddress = this.formatAddress(address, false);
+      return accountAddress === targetAddress;
+    });
+    return multisigAccount;
+  }
+
+  public updateMultisigName(newName: string): void {
+    const addressMST = this.formatAddress(this.account?.pair?.address) ?? '';
```
Comment (Nikita-Polyakov) [inline packages/sdk/src/apiAccount.ts:844]: Shouldn't here be:
`this.accountStorage?.get('MSTAddress')`

Because in `createMST` method you saved MST acc address to accountStorage?

File: packages/sdk/src/apiAccount.ts [hunk 2]
```diff
@@ -813,4 +813,43 @@ export class ApiAccount<T = void> extends WithAccountHistory implements ISubmitE
...
+    const multisigAccount = this.getMstAccount(addressMST);
+    if (multisigAccount) {
+      const pair = keyring.getPair(multisigAccount.address);
```
Comment (Nikita-Polyakov) [inline packages/sdk/src/apiAccount.ts:847]: You can use existed method for rename

`this.changeAccountName(multisigAccount.address, newName);`

**Review by stefashkaa — APPROVED** [review]

**Review by Cre-eD — APPROVED** [review]
